tensorflow_addons/image/transform_ops.py

```diff
@@ -291,3 +292,55 @@ def _image_projective_transform_grad(op, grad):
         output_shape=tf.shape(image_or_images)[1:3],
         interpolation=interpolation)
     return [output, None, None]
+
+
+@tf.function
+def rotate(images, angles, interpolation="NEAREST", name=None):
+    """Rotate image(s) counterclockwise by the passed angle(s) in radians.
+
+    Args:
+      images: A tensor of shape (num_images, num_rows, num_columns, num_channels)
+        (NHWC), (num_rows, num_columns, num_channels) (HWC), or
+        (num_rows, num_columns) (HW). The rank must be statically known (the
+        shape is not `TensorShape(None)`.
+      angles: A scalar angle to rotate all images by, or (if images has rank 4)
+        a vector of length num_images, with an angle for each image in the batch.
+      interpolation: Interpolation mode. Supported values: "NEAREST", "BILINEAR".
+      name: The name of the op.
+
+    Returns:
+      Image(s) with the same type and shape as `images`, rotated by the given
+      angle(s). Empty space due to the rotation will be filled with zeros.
+
+    Raises:
+      TypeError: If `image` is an invalid type.
+    """
+    with ops.name_scope(name, "rotate"):
```

Comment:
Please use tf.name_scope. We should always try to use tf public API if possible.

Reply:
done

Reply:
I ended up reverting this back because the function signature is different for `tf.name_scope` and `ops.name_scope`, and the other image transform functions use the `ops.name_scope` version.  I think we should EITHER change the behavior of ALL functions in this module or leave it as-is with `ops.name_scope` for consistency.

Reply:
done